### PR TITLE
Add `--legacy` flag, for compatibility with Xcode 16

### PR DESCRIPTION
**Pull request by gemmakbarlow**
Some related discussion can be seen here - https://github.com/fastlane/fastlane/issues/22132

### CODE REVIEW

**Comment by gemmakbarlow** [comment]
I welcome any tips / feedback about how to run tests and confirm the adjustments here operate correctly !

File: src/main.ts [hunk 1]
```diff
@@ -160,6 +160,7 @@ async function mergeResultBundle(
   const args = ['xcresulttool', 'merge']
     .concat(inputPaths)
     .concat(['--output-path', outputPath])
+    .concat(['--legacy'])
```
Comment (gemmakbarlow) [inline src/main.ts]: Probably needs a version check added (i.e. Xcode 16 and above); thoughts on how to do that appreciated !

**Comment by moyerr** [comment]
Hi @gemmakbarlow thanks for opening this! I ran into the same issue myself, and after digging around, it seems like @jozefizso has [a fork that's more actively maintained](https://github.com/slidoapp/xcresulttool). Would you consider opening this same PR over in that repo? If not, I'd be happy to do it 🙂

**Comment by gemmakbarlow** [comment]
> Some related discussion can be seen here - [fastlane/fastlane#22132](https://github.com/fastlane/fastlane/issues/22132)

Absolutely - thanks for finding the fork. Opened as https://github.com/jozefizso/xcresulttool/pull/1. Suggestions for testing welcomed if you know any Typescript.

**Comment by gemmakbarlow** [comment]
> > Some related discussion can be seen here - [fastlane/fastlane#22132](https://github.com/fastlane/fastlane/issues/22132)
> 
> Absolutely - thanks for finding the fork. Opened as [jozefizso#1](https://github.com/jozefizso/xcresulttool/pull/1). Suggestions for testing welcomed if you know any Typescript.

Whoops, I see I picked the wrong fork to PR against. Trying again.

**Comment by gemmakbarlow** [comment]
Closing in favor of https://github.com/slidoapp/xcresulttool/pull/3

**Pull request closed without merging**